README.md

```diff
@@ -25,50 +25,119 @@ forward, reverse, and higher-order differentiation, as well as batching using
+> Only the CPU-enabled build of jax-finufft is available as a binary from
+> conda-forge. For a GPU-enabled build, you'll need to build from source as
+> described below.
+
+To install using [mamba](https://github.com/mamba-org/mamba) (or
+[conda](https://docs.conda.io)), run:
 
 ```bash
-conda create -n jax-finufft -c conda-forge python=3.10 numpy scipy fftw cxx-compiler
-conda activate jax-finufft
-export CPATH=$CONDA_PREFIX/include:$CPATH
-python -m pip install "jax[cpu]"
-python -m pip install .
+mamba install -c conda-forge jax-finufft
 ```
 
-The `CPATH` export is needed so that the build can find the headers for libraries like FFTW installed through conda.
+### Install from source
+
+#### Dependencies
 
-For a GPU build, while the CUDA libraries and compiler are nominally available through conda,
-our experience trying to install them this way suggests that the "traditional"
-way of obtaining the [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) directly
-from NVIDIA may work best (see [related advice for Horovod](https://horovod.readthedocs.io/en/stable/conda_include.html)). After installing the CUDA Toolkit, one can set up the rest of the dependencies with:
+Unsurprisingly, a key dependency is JAX, which can be installed following the
+directions in [the JAX
+documentation](https://jax.readthedocs.io/en/latest/installation.html). If
+you're going to want to run on a GPU, make sure that you install the appropriate
+JAX build.
+
+The non-Python dependencies that you'll need are:
+
+- [FFTW](https://www.fftw.org),
+- [OpenMP](https://www.openmp.org) (for CPU, optional),
+- CUDA (for GPU, >= 11.8), and
+- cuDNN (for GPU).
+
+Older versions of CUDA may work, but they are untested.
+
+Below we provide some example workflows for installing the required dependencies:
+
+<details>
+<summary>Install CPU dependencies with mamba or conda</summary>
 
 ```bash
-conda create -n gpu-jax-finufft -c conda-forge python=3.10 numpy scipy fftw 'gxx<12'
-conda activate gpu-jax-finufft
-export CPATH=$CONDA_PREFIX/include:$CPATH
-export CMAKE_ARGS="-DCMAKE_CUDA_ARCHITECTURES=70 -DJAX_FINUFFT_USE_CUDA=ON"
+mamba create -n jax-finufft -c conda-forge python jax fftw cxx-compiler
+mamba activate jax-finufft
+```
+</details>
+
+<details>
+<summary>Install GPU dependencies with mamba or conda</summary>
+
+For a GPU build, while the CUDA libraries and compiler are nominally available
+through conda, our experience trying to install them this way suggests that the
+"traditional" way of obtaining the [CUDA
+Toolkit](https://developer.nvidia.com/cuda-downloads) directly from NVIDIA may
+work best (see [related advice for
+Horovod](https://horovod.readthedocs.io/en/stable/conda_include.html)). After
+installing the CUDA Toolkit, one can set up the rest of the dependencies with:
+
+```bash
+mamba create -n gpu-jax-finufft -c conda-forge python numpy scipy fftw 'gxx<12'
+mamba activate gpu-jax-finufft
+export CMAKE_PREFIX_PATH=$CONDA_PREFIX:$CMAKE_PREFIX_PATH
```

Comment:
Did we decide to remove this from the default instructions?

Reply:
I don't know whether or not we need it for this one. The package that seems to set it automatically is `cxx-compiler`, which isn't installed here, but I haven't tested this one, so I just kept it more or less the same as what you had before.

Reply:
Oh, good call. The reason we're using `gxx<12` is because CUDA 11 only supports GCC <= 11. CUDA 12 does support GCC 12.1, but `cxx-compilers` might install a newer version... so let's leave this as-is, I agree!